db/migrate/20170508150135_add_foreign_keys.rb

```diff
@@ -0,0 +1,49 @@
+class AddForeignKeys < ActiveRecord::Migration
+  def change
+    add_index :notifications, [:subject_id, :subject_type]
+    add_index :order_details, :project_id
+    add_index :order_statuses, :parent_id
+    add_index :orders, :merge_with_order_id
+    add_index :price_group_members, :account_id
+    add_index :products, :initial_order_status_id
+
+    # The dartmouth database violates this one
+    add_foreign_key :account_users, :users
+
+    add_foreign_key :order_details, :users, column: :assigned_user_id
+    add_foreign_key :order_details, :journals
+    add_foreign_key :order_details, :order_statuses
+    add_foreign_key :order_details, :statements
+    add_foreign_key :orders, :order_imports
+    add_foreign_key :orders, :orders, column: :merge_with_order_id
+
+    #The UIC and Dartmouth databases violate this
+    add_foreign_key :orders, :users
+
+    add_foreign_key :price_group_members, :accounts
+
+    # This is in a migration, the Dartmouth database has it, the UIC
+    # one does not
+    #add_foreign_key :price_group_members, :price_groups
```

Comment:
I see this foreign key when I look at the different schemas as well as my snapshots of UIC, open, and NU, it's just a different name in some of them `sys_c008583`. Maybe we can can drop the constraint and re-add it got get it consistent.